openquake/commonlib/tests/nrml_test.py

```diff
@@ -121,3 +122,65 @@ def test_invalid(self):
             read(fname)
         self.assertIn('Could not convert imt->intensity_measure_type: '
                       "Invalid IMT: 'SA', line 8", str(ctx.exception))
+
+    def test_invalid_srcs_weights(self):
```

Comment:
I think it would be more intuitive to call this test something like `test_invalid_srcs_weights_len`, because weights correctly sum up to 1, but the number of sources is wrong.